lib/metrics/metrics.go

```diff
@@ -58,6 +58,10 @@ var (
 	WSSessionDuration  = stats.New("ws_session_duration", stats.Trend, stats.Time)
 	WSConnecting       = stats.New("ws_connecting", stats.Trend, stats.Time)
 
+	// gRPC-related
+	GRPCReqs        = stats.New("grpc_reqs", stats.Counter)
```

Comment:
removed in-favour of `count`. 

Would be good to add `count` to the default trends. Why was it made optional? Can't see it being a breaking change?

Reply:
It wasn't a breaking change, but because we had `http_reqs` already and we don't have metric filtering yet (https://github.com/loadimpact/k6/issues/1321), the output with `count` by default would have looked very bad. Try running `k6 run -i 2 --summary-trend-stats="avg,min,med,max,p(90),p(95),count" github.com/loadimpact/k6/samples/http_batch.js` for example, you'd see something like:
```
    checks.....................: 100.00% ✓ 6   ✗ 0  
    data_received..............: 35 kB   31 kB/s
    data_sent..................: 1.8 kB  1.7 kB/s
    http_req_blocked...........: avg=87.97ms  min=2.76µs   med=9.45µs   max=619.21ms p(90)=215.37ms p(95)=417.29ms count=8
    http_req_connecting........: avg=27.59ms  min=0s       med=0s       max=194.96ms p(90)=67.58ms  p(95)=131.27ms count=8
    http_req_duration..........: avg=106.6ms  min=16.64ms  med=102.02ms max=205.76ms p(90)=205.43ms p(95)=205.6ms  count=8
    http_req_receiving.........: avg=125.86µs min=55.41µs  med=99.16µs  max=316.44µs p(90)=206.42µs p(95)=261.43µs count=8
    http_req_sending...........: avg=73.67µs  min=22.39µs  med=38.75µs  max=282.35µs p(90)=140.79µs p(95)=211.57µs count=8
    http_req_tls_handshaking...: avg=51.2ms   min=0s       med=0s       max=409.63ms p(90)=122.89ms p(95)=266.26ms count=8
    http_req_waiting...........: avg=106.4ms  min=16.48ms  med=101.9ms  max=205.16ms p(90)=205.15ms p(95)=205.15ms count=8
    http_reqs..................: 8       7.226029/s
    iteration_duration.........: avg=544.42ms min=204.51ms med=544.42ms max=884.33ms p(90)=816.34ms p(95)=850.33ms count=2
    iterations.................: 2       1.806507/s
    vus........................: 1       min=1 max=1
    vus_max....................: 1       min=1 max=1
```

All of that `count=8` repetition is a bit awful... When we have metric filtering and better ([templated](https://github.com/loadimpact/k6/issues/1319#issuecomment-579462738)) end-of-test summaries, we'd likely start hiding `http_reqs` (and probably a bunch of the `http_req_*` metrics by default), unless the user has defined a threshold on them. Some things were discussed in https://github.com/loadimpact/k6/issues/1319, but we have some groundwork to do before we get to the nice bits... :disappointed: